src/humanize/locale/fr_FR/LC_MESSAGES/humanize.po

```diff
@@ -119,257 +119,247 @@ msgctxt "9 (female)"
 msgid "th"
 msgstr "e"
 
-#: src/humanize/number.py:178
+#: src/humanize/number.py:183
```

Comment:
In my own projects, I tend to save filenames but not line numbers to PO files, to avoid such diff churn. What do you think of doing that change in another PR?

Reply:
I don't make translations myself, so if they're not useful I'll take your word for it and a PR is welcome :)

Reply:
Either the line numbers get outdated, so they are useless and get ignored, or the create churn in PRs (which I could just accept but…)

So personally I find the filenames useful to be able to look up the context (source strings and even context lines from pgettext are not always enough), I tend to dislike the line numbers.

But you could open a ticket and ping translators to see their experience!